README.md

```diff
@@ -1,3 +1,9 @@
+# Archived project
+This tool has been deprecated, instead it's used `xsltproc` to perform all the changes in server.xml file.
+If you want to resume work, please contact us at https://lists.opensuse.org/archives/list/devel@lists.uyuni-project.org/ or https://gitter.im/uyuni-project/devel
```

Comment:
All lists except announce are now deprecated, GitHub discussions should be used instead
```suggestion
If you want to resume work, please contact us at [GitHub Discussions](https://github.com/uyuni-project/uyuni/discussions) or [the `devel` channel at Gitter](https://gitter.im/uyuni-project/devel)
```